remix/app/repositories/user.server.test.ts

```diff
@@ -0,0 +1,55 @@
+import type { User } from "@prisma/client";
+import { prismaMock } from "../tests/setup-test-env";
+import UserRespository from "../repositories/user.server"
+
+describe("Test user model", () => {
+    test("Create User", async () => {
```

Comment:
This should be the method name:

```suggestion
    test("updateOrCreate", async () => {
```

Reply:
Fixed.